yarn-error.log

```diff
@@ -1,33 +1,48 @@
 Arguments: 
-  /usr/bin/node /usr/bin/yarn install
+  /usr/local/Cellar/node/12.6.0/bin/node /usr/local/Cellar/yarn/1.17.3/libexec/bin/yarn.js add webpack-dev-sever
 
 PATH: 
-  /home/unepwcmc/.rvm/gems/ruby-2.6.3/bin:/home/unepwcmc/.rvm/gems/ruby-2.6.3@global/bin:/home/unepwcmc/.rvm/rubies/ruby-2.6.3/bin:/home/unepwcmc/.rvm/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/opt/bin
+  /Users/stacytalbot/.rvm/gems/ruby-2.6.3/bin:/Users/stacytalbot/.rvm/gems/ruby-2.6.3@global/bin:/Users/stacytalbot/.rvm/rubies/ruby-2.6.3/bin:/Users/stacytalbot/.rvm/bin:/usr/local/bin:/usr/bin:/bin:/usr/sbin:/sbin:/Applications/Postgres.app/Contents/Versions/latest/bin
```

Comment:
I don't know the meaning of this file, but I can read your name here :grimacing: 
Do we need this at all??